### DISCUSSION TITLE
Flutter app question

### DISCUSSION THREAD

**Discussion by Get-108** [tesla-android/issue-tracker#274] [Q&A] [answered]
Dear Michal, thank you for great solution.
Everything was installed fast and work almost correctly.
But i still don’t  understand how to activate  fullscreen mode.
If i navigate to fullscreen address it forwarding me to onboard Youtube application.
Where is mentiined any Flutter app UI? Should i operate with something like Android SDK to reach it?
Tesla S 2021.
Thank you very much for any workaround.

**Comment by mikegapinski** [comment, marked as answer]
the fullscreen app launches a YT "app" that is actually a browser with a url for the Tesla Android frontend. If it does not load the Tesla Android app for you it is indeed a DNS problem. What does the full screen browser show for you? YT website, DNS error for "www.youtu.be"(ERR_NAME_NOT_RESOLVED)? The "domain" I use for launching the fullscreen app is not valid and used by google, it just matches the regex tesla uses to detect supported websites for theater apps. After "www.youtu.be" "loads", I do a redirect on the tesla android webserver and feed my frontend

I am aware of some little networking issues in the last version; I have most of the already fixed.

**Reply by Get-108** [reply]
Mike, thank you for the prompt answer and support!
I have seen both YT site and DNS error, yes.
And where actually can I reach parameters of Flutter app?
Thank you!

**Reply by mikegapinski** [reply]
By the parameters you mean settings? Tap the version ribbon in the upper right corner of the screen when using Tesla Android.

**Comment by Get-108** [comment]
I mean this:
“ An early fullscreen frontend is now available. You can access it using a new “Go Full-Screen” button in the Flutter App or by navigating to fullscreen.app.teslaandroid.com.”

I don’t see any “fullscreen” button after red ribbon tapped (in app config)
![Uploading image.jpg…]()

**Reply by drderiv** [reply]
Tap the red bar that runs across the top of the setup screen.

**Comment by Get-108** [comment]
Already found, thanx:) but it works only in park mode, correct?
Anyway that is a great solution for me.
Thank you, Mike. Is it possible to send donats in USDT?
